### PR TITLE
fix: Popular topics cards mobile DOCS-415

### PR DESCRIPTION
**Pull request by nicklem**
Updates stylesheets to improve the display and readability of the "most popular topics" cards on the docs homepage on smaller screens.

### CODE REVIEW

**Comment by prcr** [comment]
Tested it, and looks great!

image

I added @inesgomas as a reviewer (she has context about this issue). Let's wait a day or so to see if she has feedback before merging. :raised_hands:

**Review by inesgomas — APPROVED** [review]

**Review by prcr — DISMISSED** [review]

**Comment by nicklem** [comment]
Thanks for the tip @prcr! I included the dependabot bumps and tested locally.
Awaiting final approval.

**Comment by prcr** [comment]
> I included the dependabot bumps and tested locally.

I now realized that I usually merged the Dependabot pull requests (after testing locally), but I didn't push the built files of the template at the same time. So the changes from the bumped versions would actually only be applied next time I changed something else on the template! :facepalm:

**Review by prcr — APPROVED** [review]

**Pull request merged**
